raritan/context.py

```diff
@@ -12,12 +15,18 @@ class Context(object):
     If we want a different structure for this data in the future, we could definitely consider that.
     """
 
-    def __init__(self):
-        self.data_references = {}
-        self.release_spec_name = None
-        self.flow_id = None
-        self.current_task = None
-        self.no_logging = False
+    # A place to stash data during the flow run.
+    data_references = {}
+    # The active release specification.
+    release_spec_name = None
+    # The active flow id.
+    flow_id = None
+    # The current task/step of the flow.
+    current_task = None
+    # Disable logging/output.
+    no_logging = False
+    # The namespace to find ETL settings.
+    settings_module = 'settings'
```

Comment:
Are these changes here due to cleanup from running the tests?

Reply:
There is a subtle difference between setting class variables inside __init__ (constructor) vs on the class itself: https://stackoverflow.com/questions/1537202/difference-between-variables-inside-and-outside-of-init-class-and-instanc

Since this class is meant to be a singleton, this felt appropriate and actually functioned better.